schemas/groups/propulsion.json

```diff
@@ -82,6 +65,109 @@
       "units": "Pa"
     },
 
+    "alternatorVoltage": {
+      "description": "Alternator voltage",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "V"
+    },
+
+    "engineTime": {
+      "description": "Total running time for engine",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "s"
+    },
+
+    "engineTemperature": {
+      "description": "Engine temperature",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "K"
+    },
+
+    "coolantPressure": {
+      "description": "Coolant pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "boostPressure": {
+      "description": "Engine boost (turbo, supercharger) pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "transmission": {
+      "type": "object",
+      "properties": {
+        "gear": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "enum": ["Forward", "Neutral", "Reverse", "Fault"]
+            }
+          }
+        },
+        "oilTemperature": {
+          "description": "Oil temperature",
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "units": "K"
+        },
+        "oilPressure": {
+          "description": "Oil pressure",
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "units": "Pa"
+        }
+      }
+    },
+
+    "drive": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": ["saildrive", "shaft", "outboard"]
+        },
+        "trimState": {
```

Comment:
The good thing is that Signal K is open and extensible and we should encourage people to think of developing new sensors that could detect the position and attitude of an outboard and then be the first interface that supports the new sensor, not try and second guess what may or may not be developed in the future.